src/lib/orion_websocket/constants.cpp

```diff
@@ -0,0 +1,35 @@
+/*
+*
+* Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Felipe Ortiz
+*/
+
+
+#include "constants.h"
+
+const int WSConstants::Port = 9010;
+const int WSConstants::Pooling = 50;
+const size_t WSConstants::DataSize = 128;
+// Ok, this is can be a macro, but I prefer
+// to be consistent
+const std::string WSConstants::ProtocolName = "orion-ws";
```

Comment:
well, the sub-protocol, application level protocol, is an optional part in the Javascript API. 

Having said that, wouldn't it be 'ngsiv2-json' a proper name for the sub-protocol? Maybe in the future we implement 'ngsiv2-text' or 'ngsiv2-binary' ... 

By the way, there is a registry  of subprotocols 

https://www.iana.org/assignments/websocket/websocket.xml#subprotocol-name

if we say 'orion-xxx' it can be more difficult if others decide to implement this as we have done for Orion.